django/db/migrations/writer.py

```diff
@@ -350,6 +350,8 @@ def serialize(cls, value):
         # process.
         if isinstance(value, Promise):
             value = force_text(value)
+        elif isinstance(value, LazyObject):
+            value = value.__reduce__()[1][0]
```

Comment:
I'm a fan of this pattern but I couldn't find a way to avoid depending on `LazyObject` implementation details. Suggestions welcome.